app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,160 @@
+    <string name="backup_screen_sub_text">Not: Yedekler uygulama ayarlarını içermez.</string>
+    <string name="backup_button">Uygulama Verilerini Yedekle</string>
+    <string name="restore_button">Uygulama Verilerini Geri Yükle</string>
+    <string name="backup_restore_success">Yedek başarıyla geri yüklendi!</string>
+
+    <!-- Settings Screen -->
+    <string name="settings_screen_header">Ayarlar</string>
+    <string name="display_settings_title">Görünüm</string>
+    <string name="theme_setting">Varsayılan Tema</string>
+    <string name="theme_dialog_title">Temayı Değiştir</string>
+    <string name="theme_dialog_apply_button">Uygula</string>
+    <string name="material_you_setting" translatable="false">Material You</string>
+    <string name="material_you_setting_desc">Temayı duvar kağıdınıza göre değiştirin (Yalnızca A12+)</string>
+    <string name="material_you_error">Bu özellik yalnızca Android 12 ve üstü cihazlar içindir.</string>
+    <string name="locales_setting_title">Yerel</string>
+    <string name="date_format_setting">Tarih Biçimi</string>
+    <string name="date_format_dialog_title">Tarih Biçimini Değiştir</string>
+    <string name="preferred_currency_setting">Tercih Edilen Para Birimi</string>
+    <string name="currency_dialog_title">Tercih Edilen Para Birimini Seç</string>
+    <string name="security_settings_title">Güvenlik</string>
+    <string name="app_lock_setting">Biyometrik Kilit</string>
+    <string name="app_lock_setting_desc">GreenStash'in kilidini açmak için parmak izini ya da ekran kilidini kullan</string>
+    <string name="misc_setting_title">Diğer</string>
+    <string name="license_setting">Lisans &amp; Sözleşme</string>
+    <string name="license_setting_desc">Açık kaynak lisans bilgilerini göster.</string>
+    <string name="app_info_setting">Uygulama Bilgileri</string>
+    <string name="app_info_setting_desc">Uygulama versiyonunu ve faydalı linkleri göster.</string>
+
+    <!-- OSL Screen -->
+    <string name="osl_screen_header">Açık Kaynak Kütüphaneleri</string>
+
+    <!-- About Screen -->
+    <string name="about_screen_header">Hakkında</string>
+    <string name="about_readme_title">README</string>
+    <string name="about_readme_desc">Github deposuna ve uygulamanın README'sine göz at.</string>
+    <string name="about_privacy_title">Gizlilik Sözleşmesi</string>
+    <string name="about_privacy_desc">Github'da güncel gizlilik sözleşmemizi ve kullanım koşullarını oku.</string>
+    <string name="about_gh_issue_title">Github Sorunları</string>
+    <string name="about_gh_issue_desc">Github'da bir sorun oluşturarak hataları bildirin ya da özellik talep edin.</string>
+    <string name="about_telegram_title">Telegram Grubu</string>
+    <string name="about_telegram_desc">Fikirlerinizi paylaşmak ya da sadece sohbet etmek için Telegram'da bize katılın!</string>
```

Comment:
Tests are failing due to un-escaped (`'`) in places like `Telegram'da` , please escape them properly